app/assets/stylesheets/style.scss

```diff
@@ -104,4 +110,9 @@ body {
 
 table.center {
   background-color: $side-nav-background;
+  text-align: center;
+}
+
+.card-errors {
+  color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.